src/main/java/io/codelex/classesandobjects/practice/videostore/Video.java

```diff
@@ -0,0 +1,49 @@
+package io.codelex.classesandobjects.practice.videostore;
+
+public class Video {
+    protected String title;
```

Comment:
private
un protected is redundant
`protected String title = String title`

Reply:
hey, salabots!